frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -53,7 +53,8 @@ interface Response {
     totals: Totals;
     coinNames: CoinNames;
     chartDataMissing: boolean;
-    chartData: ChartData;
+    chartDataDaily: ChartData;
+    chartDataHourly: ChartData;
```

Comment:
This doesn't seem to be used at all. What is it for?

Reply:
I guess it's for later use?

Reply:
yeah @thisconnect will use it with the timeframe buttons.